recipes/sentry-crashpad/all/conanfile.py

```diff
@@ -25,12 +25,23 @@ class SentryCrashpadConan(ConanFile):
     exports_sources = "CMakeLists.txt", "patches/*"
     generators = "cmake"
 
+    no_copy_source = True
```

Comment:
This is very exceptionally for compiled libraries

Reply:
Yes, but if source folder is located within a build folder, it will not compile on apple. CMake file for crashpad utils contains huge part of copying and generating sources which might conflict with such approach.